test/e2e/erc721/differentChainsMock.js

```diff
@@ -1,207 +0,0 @@
-const TruffleAssert = require('truffle-assertions');
```

Comment:
Let's revert it back. We don't use ERC721 but still have it in the SCs.

Reply:
Reverted the changes